src/libraries/Common/src/System/Security/Cryptography/MLDsa.cs

```diff
@@ -1179,14 +1110,12 @@ public static MLDsa ImportMLDsaPrivateSeed(MLDsaAlgorithm algorithm, ReadOnlySpa
         {
             ThrowIfNotSupported();
```

Comment:
This method is not doing a `null` check for `algorithm`. Since this PR is flipping the class to public we should try to get as much of the error handling as reasonable done.